lib/cext/preprocess.rb

```diff
@@ -8,7 +8,7 @@
 
 ARGF.each do |line|
   if line.include?('rb_scan_args')
-    puts line.gsub(/rb_scan_args\((\w+), (\w+), \"(.*)\", /) {
+    puts line.gsub(/\srb_scan_args\((\w+), (\w+), \"(.*)\", /) {
```

Comment:
It seems `_` is not a boundary character (it is part of `\w` = `[0-9A-Za-z_]` from what I understand):
```ruby
/\brb/ =~ "_rb" # => nil
/\brb/ =~ "-rb" # => 1
```
OTOH `-` is which is what we want :smiley: